site/content/en/docs/tasks/run_rayclusters.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Run A RayCluster"
+date: 2024-01-17
+weight: 6
+description: >
+  Run a RayCluster on Kueue.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [RayCluster](https://docs.ray.io/en/latest/cluster/getting-started.html).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Make sure you are using Kueue v0.6.0 version or newer and Kuberay 1.1.0 or newer.
```

Comment:
Not sure how we care that v1.1.0 is not release yet. Probably not a big deal since v1.1.0 will be out sometime in Feb